EssentialApp/EssentialApp/SceneDelegate.swift

```diff
@@ -9,6 +9,9 @@ import EssentialFeed
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 	var window: UIWindow?
+	private var navigationController: UINavigationController?
```

Comment:
You can declare the navigation controller as a `lazy var` so it doesn't need to be optional.

```suggestion
	private lazy var navigationController = UINavigationController(
			rootViewController: FeedUIComposer.feedComposedWith(
				feedLoader: makeRemoteFeedLoaderWithLocalFallback,
				imageLoader: makeLocalImageLoaderWithRemoteFallback,
				didSelectImage: { [weak self] image in
					self?.navigateToImageComments(feedImage: image)
				}))
```